solr/core/src/java/org/apache/solr/search/CacheConfig.java

```diff
@@ -116,12 +116,10 @@ public static CacheConfig getConfig(SolrConfig solrConfig, String xpath) {
   }
 
 
-  @SuppressWarnings({"unchecked"})
   public static CacheConfig getConfig(SolrConfig solrConfig, String nodeName, Map<String,String> attrs, String xpath) {
     CacheConfig config = new CacheConfig();
     config.nodeName = nodeName;
-    @SuppressWarnings({"rawtypes"})
-    Map attrsCopy = new LinkedHashMap<>(attrs.size());
+    Map<String,String> attrsCopy = new LinkedHashMap<>(attrs.size());
     for (Map.Entry<String, String> e : attrs.entrySet()) {
       attrsCopy.put(e.getKey(), String.valueOf(e.getValue()));
```

Comment:
This would be different when value is null, so I didn't want to change it. Might not matter, but I figured it's ok to leave the extra method call for now.